### ISSUE TITLE
Official Database Data

### ISSUE THREAD

**Issue by SoaringChris** [DieselTech/CBL-ReadingLists#84]
Is it possible to have official ID data from the Marvel API included in the cbl files?

I'm wanting to use this data to power a reading tracking app that will allow you to instantly open issues via deep link to Marvel Unlimited but without actual refrences to the Marvel API it's not possible to do this without having to check each issue individually

**Comment by flips22** [comment]
yeah that would be great! @themadman0980 actually was putting together a draft in which we would have comicvine as well as GCD/comics.org ids in the xml.  In the database tag we could add marvel to be another ID side by side with cv and gcd. The nice thing is that we can add extra stuff and not break the apps that currently use the CBL files.  Ideally we would all agree on a new format, but I think if we have the data in the xml we can easily convert to a new format down the road.

I'm very interested in this project and would love to help out.

**Comment by DieselTech** [comment]
As flips said, this is something we have been considering for a while now. We want to include as many different ways as possible for people to interact with the data. Our only roadblock right now is getting a standard together that will have some staying power and get adopted by the masses.

**Comment by flips22** [comment]
@SoaringChris I haven't integrated the Marvel API into my cbl script, so I don't have an update on this, but we have a discord now, so if you want to join there and talk about your project.  We are working on a machine readable JSON version so your app could interface direction with that once we get the marvel IDs added.
